js/index.js

```diff
@@ -26,17 +43,56 @@ function removeProduct(event) {
   const target = event.currentTarget;
   console.log('The target in remove is:', target);
   //... your code goes here
+  const parentTR = target.parentNode.parentNode;
+  const parentTBODY = parentTR.parentNode;
+
+  parentTBODY.removeChild(parentTR);
+  calculateAll();
 }
 
 // ITERATION 5
 
 function createProduct() {
   //... your code goes here
+  console.log('createProduct');
+
+  let productNameDom = document.querySelector(
+    '.create-product input[type="text"]'
+  );
+  let productValDom = document.querySelector(
+    '.create-product input[type="number"]'
+  );
+
+  let tr = document.createElement('tr');
+  tr.classList.add('product');
+  tr.innerHTML = `
+            <td class="name"><span></span></td>
+            <td class="price">$<span></span></td>
+            <td class="quantity"><input type="number" value="0" min="0" placeholder="Quantity" /></td>
+            <td class="subtotal">$<span>0</span></td>
+            <td class="action"><button>Remove</button></td>`;
+  tr.querySelector('button').classList.add('btn', 'btn-remove');
+  tr.querySelector('button').addEventListener('click', removeProduct);
+
+  tr.querySelector('.name span').innerText = productNameDom.value;
+  tr.querySelector('.price span').innerText = productValDom.value;
```

Comment:
Great job for this one as well. Just to offer you an additional way to do it. You can fist define two variables somewhere before line 68:
```
  let newProductName = productNameDom.value;
  let newProductPrice = productValDom.value;
```
And then when creating the innerHTML, you refer to them as variables:
```
  tr.innerHTML = `
            <td class="name"><span>${newProductName}</span></td>
            <td class="price">$<span>${newProductPrice}</span></td>
            <td class="quantity"><input type="number" value="0" min="0" placeholder="Quantity" /></td>
            <td class="subtotal">$<span>0</span></td>
            <td class="action"><button>Remove</button></td>`;
```